src/utils/processSystem.cpp

```
@@ -1,32 +1,42 @@
 
 }
-ProcessSystem::ProcessSystem(uint threadNumber)
+ProcessSystem::ProcessSystem(uint _threadNumber) : threadNumber(_threadNumber), isRun(false)
 {
-    ProcessSystem::threadNumber = threadNumber;
-    ProcessSystem::isRun = false;
+
 }
 
-void ProcessSystem::init(vector<path> task, uint maxFailCount = 100)
+void ProcessSystem::init(vector<path> task, uint _maxFailCount)
```

cppcheck

[performance] Function parameter 'task' should be passed by const reference.

(passedByValue)